src/commonTest/kotlin/fr/acinq/lightning/channel/InteractiveTxTestsCommon.kt

```diff
@@ -846,39 +851,31 @@ class InteractiveTxTestsCommon : LightningTestSuite() {
     }
 
     @Test
-    fun `swap-in input missing user key`() {
-        val f = createFixture(100_000.sat, listOf(150_000.sat), 0.sat, listOf(), FeeratePerKw(2500.sat), 330.sat, 0)
-        val alice0 = InteractiveTxSession(f.channelKeysA, f.keyManagerA.swapInOnChainWallet, f.fundingParamsA, 0.msat, 0.msat, f.fundingContributionsA)
-        val bob0 = InteractiveTxSession(f.channelKeysB, f.keyManagerB.swapInOnChainWallet, f.fundingParamsB, 0.msat, 0.msat, f.fundingContributionsB)
+    fun `swap-in input missing user nonce`() {
```

Comment:
This is a nice test, but it shouldn't replace the previous test, it should just be a new one? We're still supporting legacy swap-ins, so we want to keep the `swap-in input missing user key` test.

Reply:
Yes I'm adding tests with mixed legacy/new swap-in outputs.

Reply:
done in https://github.com/ACINQ/lightning-kmp/pull/563/commits/02d89fb252f52edd27bbbcc42292440b503214b7

Reply:
It still looks like you deleted the previous test? I think it should be restored, it is useful to test the scenario where one side "forgets" to send the right TLV to advertise that an input is using swap-in-potentiam.

Reply:
Sorry I missed this, restored in https://github.com/ACINQ/lightning-kmp/pull/563/commits/962efbb24c9e739750ac5b31d31f642cd454abc9